osu.Desktop/Program.cs

```diff
@@ -25,17 +26,22 @@ public static class Program
         public static int Main(string[] args)
         {
             LegacyFilesystemReader.Register();
+            
+            var cwd = Directory.GetCurrentDirectory();
 
             using (DesktopGameHost host = Host.GetSuitableHost(@"osu", true))
             {
                 if (!host.IsPrimaryInstance)
                 {
                     var importer = new BeatmapImporter(host);
-
+                    Directory.SetCurrentDirectory(cwd);

```

Comment:
DesktopGameHost changes the cwd to the path of the binary. This changes it back so relative paths make sense when passed at the command line.